Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -20,4 +20,12 @@ def determine_rpath_paths
     paths += homebrew_extra_library_paths
     paths.to_path_s
   end
+
+  def determine_dynamic_linker_path(formula)
+    if formula && ["glibc", "glibc25"].include?(formula.name)
```

Comment:
It was a good idea though, that could maybe be requested one day. In particular, `binutils` is installed before `glibc` is installed, so we bottle it on CentOS 5 rather than on our usual Ubuntu 14. See here https://github.com/Linuxbrew/brew/wiki/Build-a-portable-bottle
It'd be great if it were as easy as:
```ruby
depends_on "glibc@2.5" if build.bottle?
```
That may take a fair bit of work though.